spec/rspec_puppet_facts_spec.rb

```diff
@@ -846,32 +794,28 @@
       subject do
         on_supported_os(
           supported_os: [
-            { 'operatingsystem' => 'CentOS', 'operatingsystemrelease' => %w[7] },
-            { 'operatingsystem' => 'OpenSuSE', 'operatingsystemrelease' => %w[42] },
+            { 'operatingsystem' => 'CentOS', 'operatingsystemrelease' => %w[9] },
+            { 'operatingsystem' => 'Debian', 'operatingsystemrelease' => %w[12] },
```

Comment:
getting this to pass with opensuse was ugly because sometimes it's OpenSuSE vs OpenSUSE and we don't have that many factsets, so I switched to Debian.